virt/virt-2-5-release-notes.adoc

```diff
@@ -0,0 +1,204 @@
+[id="virt-2-5-release-notes"]
+= {RN_BookName}
+include::modules/virt-document-attributes.adoc[]
+:context: virt-release-notes
+toc::[]
+
+== About Red Hat {VirtProductName}
+
+Red Hat {VirtProductName} {VirtVersion} is supported for use on {product-title} 4.6 clusters. {VirtProductName} enables you to bring traditional virtual machines (VMs) into {product-title} where they run alongside containers, and are managed as native Kubernetes objects.
+
+//Branding-Logo
+{VirtProductName} is represented by the image:Operator_Icon-OpenShift_Virtualization-5.png[{VirtProductName}] logo.
```

Comment:
@sjhala-ccs @aburdenthehand - this image doesn't exist. Can you add this image or remove this line?